contentcuration/contentcuration/static/js/edit_channel/tree_edit/hbtemplates/content_list_item.handlebars

```diff
@@ -1,84 +1,92 @@
 <div> <!-- Added to make jquery sortable more robust -->
-<input type="checkbox" id="item_{{node.id}}" class="list_item_wrapper" {{#if checked}}checked{{/if}}/>
-{{#if isfolder}}
-	<label class="list_item_wrapper folder collapsed open_folder" for="folder_{{node.id}}" title="{{node.title}}">
-{{else}}
-	<label class="list_item_wrapper file" for="item_{{node.id}}" title="{{node.title}}">
-{{/if}}
-		<div class="options pull-right hide_on_drag">
-			{{#if edit_mode}}
-				<a class="btn btn-default content-options-dropdown" role="button" rel="popover">
-					<i class="material-icons">more_horiz</i>
-				</a>
-				<br/>
-				{{#if isfolder}}<i class="folder_open_icon open_folder material-icons rtl-flip">chevron_right</i>{{/if}}
-			{{else}}
-				{{#if isfolder}}<i class="material-icons edit_item_button" title="{{formatMessage (intlGet 'messages.view_topic')}}">pageview</i>{{/if}}
-				<i class="material-icons copy_item_button" title="{{formatMessage (intlGet 'messages.copy_to_clipboard')}}">content_paste</i>
-			{{/if}}
+<label class="list_item_wrapper {{#if isfolder}}folder collapsed{{else}}file{{/if}}" for="{{#if isfolder}}folder{{else}}item{{/if}}_{{node.id}}" title="{{node.title}}">
+	<div class="options text-right hide_on_drag header_options">
+		{{#if edit_mode}}
+			<i class="content-options-dropdown material-icons" role="button" rel="popover">more_horiz</i>
```

Comment:
fyi, the `i` element is deprecated in HTML5, in favor of `em` ("emphasized").  the reason being that `i` is a stylistic descriptor ("italicized"), where as `em` is a functional descriptor.

Reply:
(I don't think this is something to worry about in this PR though!)

Reply:
Ah interesting. There are quite a number of places where the i tag is used, so this will probably need to be in another PR